examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -1647,25 +1647,24 @@ server cluster IasZone = 1280 {
   readonly attribute int8u zoneId = 17;
   readonly global attribute int16u clusterRevision = 65533;
 
-  request struct ZoneEnrollRequestRequest {
+  request struct ZoneEnrollResponseRequest {
+    IasEnrollResponseCode enrollResponseCode = 0;
+    INT8U zoneId = 1;
+  }
+
+  response struct ZoneEnrollRequest {
     IasZoneType zoneType = 0;
     INT16U manufacturerCode = 1;
   }
 
-  request struct ZoneStatusChangeNotificationRequest {
+  response struct ZoneStatusChangeNotification {
```

Comment:
Nothing uses ZoneStatusChangeNotification. 

Also being a response struct should probably end with `Response`

Reply:
It is hard for me to figure out if this is correct or not - based on the output of the IDL, this is invalid data, so either IDL generation is incomplete (this was my assumption) or input xml is wrong (great, IDL found a bug).

so if XML is wrong and provides unnecessary data, please remove from xml.
If IDL generation is incomplete, please fix IDL.

Reply:
<command source="server" code="0x00" name="ZoneStatusChangeNotification" optional="false" cli="zcl ias-zone sc"> has command source as server so according to matter spec it will not be a request